dart/lib/src/protocol/sentry_request.dart

```diff
@@ -53,17 +53,26 @@ class SentryRequest {
 
   final Map<String, String>? _other;
 
+  @Deprecated('Will be removed in v7')
   Map<String, String> get other => Map.unmodifiable(_other ?? const {});
 
+  /// The fragment of the request URL.
+  final String? fragment;
+
+  /// The inferred content type of the request payload.
+  final String? inferredContentType;
+
   SentryRequest({
     this.url,
     this.method,
     this.queryString,
     this.cookies,
+    this.fragment,
+    this.inferredContentType,
     dynamic data,
     Map<String, String>? headers,
     Map<String, String>? env,
-    Map<String, String>? other,
+    @Deprecated('Will be removed in v7.') Map<String, String>? other,
```

Comment:
Can we also add that you should use `Sentry.configureScope -> scope.setContexts(...)`.